exercises/03-sagas/after/Divergent.Shipping/Sagas/ShippingSagaData.cs

```diff
@@ -5,17 +5,15 @@ namespace Divergent.Shipping.Sagas
 {
     class ShippingSagaData : ContainSagaData
     {
-        public virtual int OrderId { get; set; }
-        public virtual int CustomerId { get; set; }
-
-        public virtual bool IsPaymentProcessed { get; set; }
-        public virtual bool IsOrderSubmitted { get; set; }
-
-        public virtual ICollection<Product> Products { get; set; }
+        public int OrderId { get; set; }
+        public int CustomerId { get; set; }
+        public bool IsOrderSubmitted { get; set; }
+        public bool IsPaymentProcessed { get; set; }
+        public ICollection<Product> Products { get; set; }
```

Comment:
I'd prefer to have the whitespace alteration and re-ordering of properties (and in other similar places) in a separate commit. They are unrelated changes, and also make it more difficult to spot a typo.

This is a minor point though, and I wouldn't block the PR on it.